packages/website/markdown-plugins/rehype-nlds-components/figure.ts

```diff
@@ -0,0 +1,14 @@
+import type { Element } from 'hast';
+import { isElement } from 'hast-util-is-element';
+import { selectAll } from 'hast-util-select';
+import { addClassName } from './_add-class-name.ts';
+
+export function figure(element: Element) {
+  if (isElement(element, 'figure') === false) return;
+
+  addClassName('utrecht-figure', element);
+
+  selectAll('img', element).forEach(addClassName('utrecht-img'));
```

Comment:
Ik zie in de HTML output een extra class name `utrecht-img--scale-down`, die ik hier niet zie staan. Just wondering: is deze code voor `utrecht-img` dan wel nodig, als `utrecht-img` al door andere code geregeld lijkt te worden?

```html
<figure class="utrecht-figure">
    <img src="https://raw.githubusercontent.com/nl-design-system/gebruikersonderzoeken/assets/vng-online-formulieren__vinden-van-formulier.png" class="utrecht-img utrecht-img--scale-down">
    <figcaption class="utrecht-figure__caption">Figuur 1. Scherm met contactmogelijkheden voor gemeente Voorbeeld</figcaption>
</figure>
```

Reply:
klinkt als een robuust plan!